### PR TITLE
Altda devnet monitoring

### PR DESCRIPTION
**Pull request by samlaf**
This doesn't work right now because rebased on top of op 1.9.3, which has a bug: https://github.com/ethereum-optimism/optimism/issues/12354

Until this is fixed, I cherry-picked the commits here on top of a previous release. Use the altda-devnet-monitoring-1.9.2 branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +27,37 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+## devnet with eigenda alt-da
+
+First you will need to build the dev docker image for eigenda-proxy, until we make a new release.
```
Comment (samlaf) [inline README.md:32]: TODO: change this after we release the new proxy

**Comment by tynes** [comment]
Do you have plans on migrating to kurtosis? We would like to delete the docker compose devnet soon and rely strictly on kurtosis

**Comment by samlaf** [comment]
> Do you have plans on migrating to kurtosis? We would like to delete the docker compose devnet soon and rely strictly on kurtosis

@tynes yes sir! but there's still a bunch of stuff missing for me to use kurtosis, and don't have time to fix them right now. Plus I still have this PR open that needs to be unblocked by you guys https://github.com/ethpandaops/optimism-package/pull/69

For now we just want to use this devnet to make a high-throughput eigenda x op-stack announcement. We don't plan on supporting it or using it long term. Will try to migrate to kurtosis slowly for future projects. But for now would be super useful if we can fix the issue mentioned in the description so we can make this announcement.

**Comment by tynes** [comment]
It should be a simple fix, did you see my comment on the monorepo?

File: README.md (hunk 1 above)
Comment (bxue-l2) [reply README.md:32]: made https://github.com/Layr-Labs/eigenda-proxy/releases/tag/v1.5.0

File: README.md [hunk 2]
```diff
@@ -26,6 +27,32 @@
...
+
+```bash
+git clone https://github.com/Layr-Labs/eigenda-proxy.git
+make docker-build
+```
+
+Now create a `.env` file in the root of this repository with the following content (you will need to add an ecdsa private key that is authorized on the eigenda disperser, as well as an ethereum rpc endpoint):
+
+```bash
+MEMSTORE_ENABLED=false
```
Comment (bxue-l2) [inline README.md:42]: update the flag to EIGENDA_PROXY_MEMSTORE_ENABLED

File: cannon/README.md [hunk 3]
```diff
@@ -29,8 +29,8 @@ cd ../cannon
 make cannon
 
 # Transform MIPS op-program client binary into first VM state.
-# This outputs state.json (VM state) and meta.json (for debug symbols).
-./bin/cannon load-elf --type singlethreaded --path=../op-program/bin/op-program-client.elf
+# This outputs state.bin.gz (VM state) and meta.json (for debug symbols).
+./bin/cannon load-elf --type singlethreaded-2 --path=../op-program/bin/op-program-client.elf
```
Comment (bxue-l2) [inline cannon/README.md:33]: why change into bin.gz?

File: cannon/mipsevm/tests/evm_common_test.go [hunk 4]
```diff
@@ -315,6 +315,69 @@ func TestEVMSingleStep_LoadStore(t *testing.T) {
 	}
 }
 
+func TestEVMSingleStep_MovzMovn(t *testing.T) {
```
Comment (bxue-l2) [inline cannon/mipsevm/tests/evm_common_test.go:318]: what is the reason to change EVM test?

File: cannon/README.md (hunk 3 above)
Comment (samlaf) [reply cannon/README.md:33]: ehhh this PR is prob just targeting the wrong commit. :\

File: cannon/mipsevm/tests/evm_common_test.go (hunk 4 above)
Comment (samlaf) [reply cannon/mipsevm/tests/evm_common_test.go:318]: ehhh this PR is prob just targeting the wrong commit. :\
I didn't change this.

**Pull request closed without merging**